barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/ultra_recursive_verifier.cpp

```diff
@@ -52,14 +52,14 @@ std::array<typename Flavor::GroupElement, 2> UltraRecursiveVerifier_<Flavor>::ve
     VerifierCommitments commitments{ key };
     CommitmentLabels commitment_labels;
 
-    const auto circuit_size = transcript->template receive_from_prover<FF>("circuit_size");
-    const auto public_input_size = transcript->template receive_from_prover<FF>("public_input_size");
-    const auto pub_inputs_offset = transcript->template receive_from_prover<FF>("pub_inputs_offset");
+    transcript->template receive_from_prover<FF>("circuit_size");
+    transcript->template receive_from_prover<FF>("public_input_size");
+    transcript->template receive_from_prover<FF>("pub_inputs_offset");
 
     // For debugging purposes only
-    ASSERT(static_cast<uint32_t>(circuit_size.get_value()) == key->circuit_size);
-    ASSERT(static_cast<uint32_t>(public_input_size.get_value()) == key->num_public_inputs);
-    ASSERT(static_cast<uint32_t>(pub_inputs_offset.get_value()) == key->pub_inputs_offset);
+    // ASSERT(static_cast<uint32_t>(circuit_size.get_value()) == key->circuit_size);
+    // ASSERT(static_cast<uint32_t>(public_input_size.get_value()) == key->num_public_inputs);
+    // ASSERT(static_cast<uint32_t>(pub_inputs_offset.get_value()) == key->pub_inputs_offset);
```

Comment:
```suggestion
    ASSERT(static_cast<uint32_t>(circuit_size.get_value()) == key->circuit_size);
    ASSERT(static_cast<uint32_t>(public_input_size.get_value()) == key->num_public_inputs);
    ASSERT(static_cast<uint32_t>(pub_inputs_offset.get_value()) == key->pub_inputs_offset);
```
Should we bring these back?

Reply:
no, they don't work anymore since the proof is garbage when the witnesses are not valid

Reply:
Then let's delete or pass in the has_valid_witness_params field